braindecode/models/deep4.py

```diff
@@ -308,24 +319,27 @@ def add_conv_pool_block(
                 init.constant_(bnorm_weight, 1)
                 init.constant_(bnorm_bias, 0)
 
-        init.xavier_uniform_(self.conv_classifier.weight, gain=1)
-        init.constant_(self.conv_classifier.bias, 0)
+        init.xavier_uniform_(self.final_layer[0].weight, gain=1)
```

Comment:
```suggestion
        init.xavier_uniform_(self.final_layer.conv_classifier.weight, gain=1)
```
Let's also adjust here to use explicit layer names